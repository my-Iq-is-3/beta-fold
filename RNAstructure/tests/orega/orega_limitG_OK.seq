>RA7680	Mean Basepair Probability: 0.2554		Complexity: 0.826
GGGGGCGUAGCUCAGAuGGUAGAGCGCUCGCUUgGCgUGUGACAGAUAAAGGGAUCGaUACCCGGCGCCUCCACCA