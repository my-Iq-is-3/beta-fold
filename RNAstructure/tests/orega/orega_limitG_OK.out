Initializing mutant RNA sequences...		DONE
Running OREGA calculations:
	ITERATION 1
Sequence 1:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.0033 	Mean Basepair Probability = 0.6634	Complexity = 0.667 	 Basepair fraction: 0.70 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1741 	Mean Basepair Probability = 0.4926	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 2
Sequence 1:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.0255 	Mean Basepair Probability = 0.6411	Complexity = 0.667 	 Basepair fraction: 0.65 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1741 	Mean Basepair Probability = 0.4926	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 3
Sequence 1:	 Score = 1.0255 	Mean Basepair Probability = 0.6411	Complexity = 0.667 	 Basepair fraction: 0.65 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.0790 	Mean Basepair Probability = 0.5877	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 4
Sequence 1:	 Score = 1.1738 	Mean Basepair Probability = 0.4929	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1613 	Mean Basepair Probability = 0.5054	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.0565 	Mean Basepair Probability = 0.6102	Complexity = 0.667 	 Basepair fraction: 0.65 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 5
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1741 	Mean Basepair Probability = 0.4926	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1741 	Mean Basepair Probability = 0.4926	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 6
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 7
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1130 	Mean Basepair Probability = 0.5537	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 8
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 9
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.0682 	Mean Basepair Probability = 0.5984	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.0695 	Mean Basepair Probability = 0.5972	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 10
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4897	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1496 	Mean Basepair Probability = 0.5171	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1054 	Mean Basepair Probability = 0.5612	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 11
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4897	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1769 	Mean Basepair Probability = 0.4898	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1054 	Mean Basepair Probability = 0.5612	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 12
Sequence 1:	 Score = 1.1769 	Mean Basepair Probability = 0.4897	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1769 	Mean Basepair Probability = 0.4897	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 13
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1114 	Mean Basepair Probability = 0.5552	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 14
Sequence 1:	 Score = 1.1335 	Mean Basepair Probability = 0.5332	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1488 	Mean Basepair Probability = 0.5179	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 15
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.0972 	Mean Basepair Probability = 0.5694	Complexity = 0.667 	 Basepair fraction: 0.65 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 16
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1158 	Mean Basepair Probability = 0.5509	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 17
Sequence 1:	 Score = 1.0802 	Mean Basepair Probability = 0.5864	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1372 	Mean Basepair Probability = 0.5295	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1114 	Mean Basepair Probability = 0.5552	Complexity = 0.667 	 Basepair fraction: 0.55 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1552 	Mean Basepair Probability = 0.5114	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 18
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 19
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
	ITERATION 20
Sequence 1:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 2:	 Score = 1.1367 	Mean Basepair Probability = 0.5300	Complexity = 0.667 	 Basepair fraction: 0.50 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 3:	 Score = 1.1935 	Mean Basepair Probability = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Sequence 4:	 Score = 1.0663 	Mean Basepair Probability = 0.6004	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC

Output:   
Final Sequence 1:	 Score = 1.1935 	Mean Basepair Probabilty = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Final Sequence 2:	 Score = 1.1935 	Mean Basepair Probabilty = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Final Sequence 3:	 Score = 1.1935 	Mean Basepair Probabilty = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
Final Sequence 4:	 Score = 1.1935 	Mean Basepair Probabilty = 0.4732	Complexity = 0.667 	 Basepair fraction: 0.60 	 Optimized Segment : CUUgGCgUGUGAGAGGUACC
